lib/at_cmd_parser/src/at_params.c

```diff
@@ -87,6 +87,13 @@ int at_params_list_init(struct at_param_list *list,
 
 	list->param_count = max_params_count;
 
+	for (size_t i = 0; i < list->param_count;
```

Comment:
This is not needed. We are using `calloc`, this means the parameter list is already initialized.

Reply:
This seems to be comming from the file that @hmhalvorsen  zephyrified. I am not sure why he choose to do this, I can probably remove it. As you say calloc should take care of it.